climada/hazard/tc_clim_change.py

```diff
@@ -1,174 +1,375 @@
+        0.45,0.33,0.47,0.61,0.39,0.39,0.54,0.63,0.62,0.54,0.68,0.64,0.66,0.54,0.66,0.72,0.61,
+        0.64, 0.68,0.75,0.9,1.02,0.92,0.85,0.98,0.945764286,1.011064286,1.048564286,1.049564286,
+        1.070264286,1.126564286,1.195464286,1.215064286,1.246964286,1.272564286,1.262464286,
+        1.293464286,1.340864286,1.391164286,1.428764286,1.452564286,1.494164286,1.520664286,
+        1.557164286,1.633664286,1.654264286,1.693264286,1.730264286,1.795264286,1.824264286,
+        1.823864286,1.880664286,1.952864286,1.991764286,1.994764286,2.085764286,2.105764286,
+        2.155064286,2.227464286,2.249964286,2.313664286,2.341464286,2.394064286,2.457364286,
+        2.484664286,2.549564286,2.605964286,2.656864286,2.707364286,2.742964286,2.789764286,
+        2.847664286,2.903564286,2.925064286,2.962864286,3.002664286,3.069264286,3.133364286,
+        3.174764286,3.217764286,3.256564286,3.306864286,3.375464286,3.420264286,3.476464286,
+        3.493864286,3.552964286,3.592364286,3.630664286,3.672464286,3.734364286,3.789764286,
+        3.838164286,3.882264286,3.936064286,3.984064286,4.055764286,4.098964286,4.122364286,
+        4.172064286,4.225264286,4.275064286,4.339064286,4.375864286,4.408064286,4.477764286
+        ]
+
+    return gmst_info
 
-def calc_scale_knutson(ref_year=2050, rcp_scenario=45):
+def get_knutson_data():
     """
-    Comparison 2081-2100 (i.e., late twenty-first century) and 2001-20
-    (i.e., present day). Late twenty-first century effects on intensity and
-    frequency per Saffir-Simpson-category and ocean basin is scaled to target
-    year and target RCP proportional to total radiative forcing of the
-    respective RCP and year.
+    Get data to generate figures 1b to 4b in Knutson et al., (2020):
 
-    Parameters
-    ----------
-    ref_year : int, optional
-        year between 2000 ad 2100. Default: 2050
-    rcp_scenario: int, optional
-        26 for RCP 2.6, 45 for RCP 4.5. The default is 45
-        60 for RCP 6.0 and 85 for RCP 8.5.
+    Tropical cyclones and climate change assessment. Part II: Projected 
+        response to anthropogenic warming. Bull. Amer. Meteor. Soc., 101 (3), E303–E322, 
+        https://doi.org/10.1175/BAMS-D-18-0194.1.
+
+    for 4 variables (i.e., cat05 frequency, cat45 frequency, intensity, precipitation rate), 
+    6 regions (i.e., N. Atl, NW Pac., NE Pac., N. Ind, S. Ind., SW Pac.) and 
+    5 metrics (i.e., 5% or 10%, 25%, 50%, 75%, 95% or 90%).
 
     Returns
     -------
-    factor : float
-        factor to scale Knutson parameters to the give RCP and year
+    knutson_data : np.array of dimension (4x6x5)
+        array contaning data used by Knutson et al. (2020) to project changes in cat05 frequency,
+        cat45 frequency, intensity and precipitation rate (first array's dimension), for the
+        N. Atl, NW Pac., NE Pac., N. Ind, S. Ind., SW Pac. regions (second array's dimension)
```

Comment:
These are different basin names as in the rest of the code. Why? What is the relation?